index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <h6 id="title-bottom">The man who saved a billion lives</h6>
+      <div id="img-div">
+        <img src="tribute-page-main-image.jpg" id="image"/>
+        <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+      </div>
+      <h2 id="time-line">Here's a time line of Dr. Borlaug's life:</h2>
+      <div id="content">
+        <ul id="list">
+          <li>
+            <strong>1914</strong> - Born in Cresco, Iowa
+          </li>
+          <li>
+            <strong>1933</strong> - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"
+          </li>
+          <li>
+            <strong>1935</strong> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+          </li>
+          <li>
+            <strong>1937</strong> - Finishes university and takes a job in the US Forestry Service
+          </li>
+          <li>
+            <strong>1938</strong> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.
+          </li>
```

Comment:
### Semantics :

All text should be enclosed in a `p` tag instead of directly inside `li` tag.

Reply:
- [ ] It is important that for every comment you add you should as well go deep by explaining the `why` a change is being requested. This would assist the students to better understand the reason for a change and they know how best to go about it in their next projects :+1: :sparkles: 
- [ ] please i kindly suggest you try to sound polite words when requesting for changes from the student, you can use words such as `i kindly suggest, i think you can, please ensure you` :+1: :sparkles: